chapter_appendix_math/single-variable-calculus_vn.md

```diff
@@ -703,7 +706,7 @@ $$
 Let's see how this works in code and observe how increasing the degree of the Taylor approximation brings us closer to the desired function $e^x$.
 -->
 
-*dịch đoạn phía trên*
+Hãy cùng tìm hiểu điều này bằng mã nguồn và quan sát việc tăng bậc của xấp xỉ Taylor đưa ta đến gần hơn với hàm mong muốn $e^x$.
```

Comment:
```suggestion
Hãy cùng tìm hiểu cách lập trình thứ này và quan sát việc tăng bậc của xấp xỉ Taylor đưa ta đến gần hơn với hàm mong muốn $e^x$ như thế nào.
```

Reply:
Nghe chưa đúng lắm nhỉ, how bout this
```suggestion
Hãy cùng tìm hiểu điều này bằng cách lập trình và quan sát việc tăng bậc của xấp xỉ Taylor đưa ta đến gần hơn với hàm mong muốn $e^x$ như thế nào.
```

Reply:
@minhduc0711 suggestion của bạn tốt hơn đó. Lấy cái đó đi.